ext/bin/require-jdk

```diff
@@ -49,5 +50,8 @@ fi
 chmod +x "$tmpdir/install-jdk.sh"
 mkdir -p "$install"
 
-# Piping to tail to get the last line is necessary until https://github.com/sormuras/bach/issues/35 is resolved
 $tmpdir/install-jdk.sh --emit-java-home -f "$jdkver" -l "$licensetype" --target "$install/jdk"
```

Comment:
Let's quote `"$tmpdir/install-jdk.sh"` so we don't have to worry about the tmpdir chars.